.changeset/nasty-timers-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'@interledger/open-payments': minor
```

Comment:
Im trying to think of a better naming when it comes to the "experimental" nature of the change. Maybe I'm over-complicating this though